web-client/src/transaction.rs

```diff
@@ -1,10 +1,13 @@
+use nimiq_transaction::TransactionFlags;
```

Comment:
I'd move this `use` to the second block of imports where nimiq crates are listed